frontend/src/app/features/bim/ifc_models/ifc-viewer/ifc-viewer.component.ts

```diff
@@ -130,13 +132,35 @@ export class IFCViewerComponent implements OnInit, OnDestroy, AfterViewInit {
           this.ifcData.projects,
         );
       });
+
+    this.toolbarIconSubscription = this.replaceXeokitToolbarIcons();
+  }
+
+  private replaceXeokitToolbarIcons():Subscription {
```

Comment:
Please leave a comment here, explaining why all of that is necessary

Reply:
done